src/layouts/UserLayout.less

```diff
@@ -1,15 +1,20 @@
 @import "~antd/lib/style/themes/default.less";
 
-.container {
-  background: #f0f2f5;
-  background-image: url('https://gw.alipayobjects.com/zos/rmsportal/TVYTbAXWheQpRcWDaDMu.svg');
-  width: 100%;
-  min-height: 100%;
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 100%;
-  padding: 110px 0 144px 0;
-  position: relative;
+.content {
+  padding: 32px 0;
+}
+
+@media (min-width: @screen-md-min) {
```

Comment:
Why do we have to add this restriction?

Reply:
Hide the background image on phone, you just see some of the dot and the position gets a bit strange of it.

Reply:
👌